bedrock/firefox/templates/firefox/private-browsing.html

```diff
@@ -0,0 +1,169 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Firefox Web browser — Private Browsing with Tracking Protection — Mozilla') }}{% endblock %}
+{% block page_desc %}{{ _('Get more protection and the most privacy, only from Firefox. No other browser gives you so much control over your digital footprint.')}}{% endblock %}
+{% block body_id %}firefox-private-browsing{% endblock %}
+{% block body_class %}mozid{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_private_browsing' %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {{ fxfamilynav('desktop', 'private-browsing') }}
+{% endblock %}
+
+{% block content %}
+
+<main role="main">
+  <section class="intro">
+    <div class="container">
+      <header>
+        <h1>{{ _('More protection. <br>The most privacy. <br>Only from Firefox.') }}</h1>
+        <div class="download-buttons">
+          <div class="firefox-latest-cta">
+            <a href="https://www.mozilla.org" class="button-flat-dark" data-highlight="privateWindow" role="button">{{ _('Try Private Browsing') }}</a>
```

Comment:
This URL should probably fall back to something more useful than the homepage. If someone doesn't have the `privateWindow` target in their pallet, then the button will fall back to a regular link instead of highlighting the icon. A SUMO article like https://support.mozilla.org/kb/private-browsing-use-firefox-without-history/may be more appropriate.